packages/message-relayer/src/service.ts

```diff
@@ -151,6 +151,20 @@ export class MessageRelayerService extends BaseService<MessageRelayerOptions> {
       await sleep(this.options.pollingInterval)
 
       try {
+        // Check that the correct address is set in the address manager
+        const relayer = await this.state.Lib_AddressManager.getAddress(
+          'OVM_L2MessageRelayer'
+        )
+        // If it is address(0), then message relaying is not authenticated
```

Comment:
Should we throw in this case?

Reply:
If its set to `address(0)` then there is no authentication so this can operate without a problem

Reply:
Ohhh I see.